Can not deploy smart contract on local polygon edge network

# [ [ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_getCode err="unable to fetch code" ]
## Description
After polygon edge network setup on local as instructed in documents, we are trying to deploy "Hello World" solidity smart contract using Truffle with below network config in truffle.config file.

polygonEdge: {
      provider: () => new HDWalletProvider(privateKey, `http://localhost:10002/`),
      network_id: `*`,
      chain_id: 100,
      gas: 2000000
}

Getting below error on 1st bootnode : 
[ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_getCode err="unable to fetch code"



## Your environment
- OS : MacOS 12.5.
- The version of the Polygon Edge : latest.    

## Steps to reproduce
- Which commands triggered the issue, if any : 
truffle migrate --network polygonEdge

- Provide us with the content of your genesis file : 
polygon-edge genesis --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ip4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmMG7rttF8ubwHym3TSizeXu1rXv1t3tPBE3os9zvWT8Um --bootnode /ip4/127.0.0.1/tcp/20001/p2p/16Uiu2HAmTibmbex25fXrZ8FRmS4Tu3dxxD1tB6QpmjTnVuSCdCve --premine=0x31FC2eaDE2B8086d4A27B744882DCB20e5e2CE7c:1000000000000000000000 --premine=0xb1feD6B9F2781b9c39c124598632D53Ca6c97D91:1000000000000000000000 --block-gas-limit 1000000000

- Provide us with commands that you used to start your validators.
polygon-edge server --data-dir ./test-chain-1 --chain genesis.json --grpc-address :10000 --libp2p :10001 --jsonrpc :10002 --seal --price-limit 0


## Expected behavior
- Tell us what should happen.
smart contract should deploy on local network and respond with success status and contract address.

- Tell us what happened instead.
get an error on bootnode 1, after truffle migrate
[ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_getCode err="unable to fetch code"

## Logs
2022-11-30T18:49:26.306+0530 [INFO]  polygon.server.ibft.consensus: sequence done: height=888
2022-11-30T18:49:26.306+0530 [INFO]  polygon.server.ibft: canceled sequence: sequence=888
2022-11-30T18:49:26.306+0530 [INFO]  polygon.server.ibft.consensus: sequence started: height=889
2022-11-30T18:49:26.306+0530 [INFO]  polygon.server.ibft.consensus: round started: round=0
2022-11-30T18:49:26.975+0530 [ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_getCode err="unable to fetch code"
2022-11-30T18:49:27.975+0530 [ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_getCode err="unable to fetch code"
2022-11-30T18:49:28.602+0530 [INFO]  polygon.blockchain: new block: number=889 txs=0 hash=0x674c01b255e71d08f43da71999675b568766a52f76d5ba67f354da6a17535857 parent=0xa3f55b80b25dde83b499e1ad0fe2d53e36eb1cbe316771d5af9d2a23cb388892 generation_time_in_seconds=3
2022-11-30T18:49:28.611+0530 [INFO]  polygon.blockchain: block already inserted: block=889 source=consensus
2022-11-30T18:49:28.611+0530 [INFO]  polygon.server.ibft: block committed: number=889 hash=0x674c01b255e71d08f43da71999675b568766a52f76d5ba67f354da6a17535857 validation_type=bls validators=4 committed=3


## Proposed solution

--- a/server/server.go
+++ b/server/server.go
@@ -490,7 +490,7 @@ func (j *jsonRPCHub) GetCode(root types.Hash, addr types.Address) ([]byte, error
 		return nil, err
 	}
 
-	code, ok := j.state.GetCode(account.Root)
+	code, ok := j.state.GetCode(types.BytesToHash(account.CodeHash))
 	if !ok {
 		return nil, fmt.Errorf("unable to fetch code")
 	}

